crawler/crawl-device.py

```diff
@@ -387,16 +397,30 @@ def database_push(username, up_hosts, device_types, os_names, os_versions, city,
             
         except Exception as e:
             print("Error:", e)
-            
+
+'''
+Function: check_args()
+Args: None
+Checks arguments provided for a username
+Returns username field
+'''
+def check_args():
```

Comment:
nice idea, checking/validating argv and printing usage message if incorrect :+1: